core/src/main/java/de/danoeh/antennapod/core/service/playback/PlayerStatus.java

```diff
@@ -1,24 +1,33 @@
 package de.danoeh.antennapod.core.service.playback;
 
 public enum PlayerStatus {
-    INDETERMINATE,  // player is currently changing its state, listeners should wait until the player has left this state.
-	ERROR, 
-	PREPARING, 
-	PAUSED, 
-	PLAYING, 
-	STOPPED, 
-	PREPARED, 
-	SEEKING,
-	INITIALIZING,			// playback service is loading the Playable's metadata
-	INITIALIZED;	// playback service was started, data source of media player was set.
+    INDETERMINATE(0),  // player is currently changing its state, listeners should wait until the player has left this state.
+	ERROR(-1),
+	PREPARING(19),
+	PAUSED(30),
+	PLAYING(40),
+	STOPPED(5),
+	PREPARED(20),
+	SEEKING(29),
+	INITIALIZING(9),			// playback service is loading the Playable's metadata
+	INITIALIZED(10);	// playback service was started, data source of media player was set.
```

Comment:
This essentially creates a hierarchy among the different statuses. We can use it to simplify some checks.